### PR TITLE
Add support for https

### PR DESCRIPTION
**Pull request by motocombotag**
This PR adds support for `https://` based Giphy API call based on the current URL's protocol.

### CODE REVIEW

File: src/services/giphy.service.js [hunk 1]
```diff
@@ -9,7 +9,8 @@
   /* @ngInject */
   function giphyService($http, giphyConfig) {
 
-    var baseUrl = 'http://api.giphy.com/v1/gifs';
+    var baseUrl = window.location.protocol === 'https' ? 'https://api.giphy.com/v1/gifs'
```
Comment (gillchristian) [inline src/services/giphy.service.js]: What about doing this instead?

```js
var baseUrl = window.location.protocol + '://api.giphy.com/v1/gifs'
```

**Review by gillchristian — APPROVED** [review]
LGTM :rocket:

File: src/services/giphy.service.js (hunk 1 above)
Comment (motocombotag) [reply src/services/giphy.service.js]: That was deliberate to account for the URLs that start with `file://`, `ftp://`, but anyway, I made a commit.

Comment (ndelvalle) [reply src/services/giphy.service.js]: Yeah I think it might be better to always use `https`

**Comment by ndelvalle** [comment]
I think `window.location.protocol` returns something like `https:` so we will end up having double `:`. I think is maybe better to use always `https` instead of making this dynamic right? I don't se the need to ever use `http`, @gillchristian ?

**Comment by motocombotag** [comment]
Yeah, I think always using https is okay.

**Review by gillchristian — APPROVED** [review]

**Comment by gillchristian** [comment]
:shipit: like is :fire:

**Comment by ndelvalle** [comment]
@motocombotag give it at try at `ng-giphy@1.0.3` and thanks for your contribution 🙂

**Pull request merged**